### PR TITLE
Feature/61 Search result page

### PR DESCRIPTION
**Pull request by Quipex**

### CODE REVIEW

File: backend/core/src/main/java/com/knewless/core/elasticsearch/dto/RangeBoundaryType.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	 */
+	GTE,
+	/**
+	 * Less than '<'
+	 */
+	LT,
+	/**
+	 * Less than or equals '<='
+	 */
+	LTE;
```
Comment (AndersenGans) [inline backend/core/src/main/java/com/knewless/core/elasticsearch/dto/RangeBoundaryType.java:23]: possible to make one-line comments?

File: frontend/src/screens/SearchResultsPage/models/EsModels.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { EsDataType } from '@screens/Search/models/EsDataTypes';
+import { ComplexityLevel } from '@models/LevelsEnum';
+
+// backend/core/src/main/java/com/knewless/core/elasticsearch/model/EsMapper.java
```
Comment (AndersenGans) [inline frontend/src/screens/SearchResultsPage/models/EsModels.ts:4]: required?

File: frontend/src/store.ts [hunk 3]
```diff
@@ -10,7 +10,9 @@ declare global {
   }
 }
 
-const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+  // eslint-disable-next-line no-mixed-operators
```
Comment (AndersenGans) [inline frontend/src/store.ts:14]: required?

File: backend/core/src/main/java/com/knewless/core/elasticsearch/dto/RangeBoundaryType.java (hunk 1 above)
Comment (Quipex) [reply backend/core/src/main/java/com/knewless/core/elasticsearch/dto/RangeBoundaryType.java:23]: No. It's not displayed as javadoc then. 
example

File: frontend/src/screens/SearchResultsPage/models/EsModels.ts (hunk 2 above)
Comment (Quipex) [reply frontend/src/screens/SearchResultsPage/models/EsModels.ts:4]: This is a comment to look-up corresponding model on backend.

File: frontend/src/store.ts (hunk 3 above)
Comment (Quipex) [reply frontend/src/store.ts:14]: Makes life easier as it enables traces in redux-devtools (you can see where an action was triggered from)

**Review by MaximGoncharukSW — APPROVED** [review]

**Review by MaximGoncharukSW — APPROVED** [review]

**Pull request merged**